newsfragments/2507.misc.rst

```diff
@@ -0,0 +1 @@
+Deprecated manual worker commitments using the CLI.
```

Comment:
This could be a good candidate for the "removal" category for newsfragments that applies to deprecations and removals - see "pyproject.toml" i.e. `2507.removal.rst`

Reply:
:heavy_check_mark: